git/git.go

```diff
@@ -59,14 +59,14 @@ func LsRemote(remote, remoteRef string) (string, error) {
 		return "", errors.New("remote required")
 	}
 	if remoteRef == "" {
-		return simpleExec("git", "ls-remote", remote)
+		return subprocess.SimpleExec("git", "ls-remote", remote)
```

Comment:
I may have preferred a `github.com/github/git-lfs/lfsutil` sub package or something, but this works too :+1: